import_package.md

```diff
@@ -0,0 +1,280 @@
+# Import package
+
+* An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+* A valid import package must contain all resource files that are referenced in the 
+  **manifest.yaml**.
+
+## Meta-data
+
+* `apiVersion` is set to `v1beta1` 
+
+## Tasks
+
+* `tasks`: is a list of api actions and resource tasks.
+    * `id`: Each task must contain a unique **id** property that only contains upper-case, 
+       lower-case letters, digits and an underscore (`[A-Za-z0-9_]+`)
+    * `type`: Must be either **api** or **resource**. All required properties of the specified
+       type have to be implemented.
+    * `name` *(optional)*: A human readable name for the action to easier identify the action
+    * `context` *(optional)*: An optional context which can contain additional data for the 
+      API action or resource task. A context must only contain string values and supports templating.
+
+### API action
+An API action is a task that interacts with a specific Keptn API endpoint. The JSON results of the endpoint
+will be stored in ` .Tasks.<task-id>.Response.<field>` and are accessible for all following tasks. The following
+properties have to be implemented by an API action:
+* `action`: The name of the Keptn API endpoint that should be called. 
```

Comment:
```suggestion
* `action`: The name of the Keptn API endpoint to call. 
```
I think you mean that it will be called; "should" could imply that it's optional